packages/wallet/backend/src/user/service.ts

```diff
@@ -109,7 +118,76 @@ export class UserService implements IUserService {
 
     return !!user
   }
+  public async createDefaultAccount() {
+    const existingUser = await this.getByEmail(
+      this.env.DEFAULT_WALLET_ACCOUNT.email
+    )
 
+    if (existingUser) return
+
+    await this.rafikiClient.createAsset('USD', 2)
+    const defaultWalletUser = this.env.DEFAULT_WALLET_ACCOUNT
+    const defaultBoutiqueUser = this.env.DEFAULT_BOUTIQUE_ACCOUNT
+
+    let args = {
+      ...defaultWalletUser,
+      isEmailVerified: true
+    }
+
+    const createdWalletUser = await User.query().insertAndFetch(args)
+
+    const wallertPromis = this.accountService.createDefaultAccount(
+      createdWalletUser.id,
+      false
+    )
+
+    args = {
+      ...defaultBoutiqueUser,
+      isEmailVerified: true
+    }
+    const createdBoutiqueUser = await User.query().insertAndFetch(args)
+
+    const boutiquePromise = this.accountService.createDefaultAccount(
```

Comment:
create user + default account are similar and can be extracted in one method that will be called with default user details